src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -109,6 +108,13 @@
             "<li>TN - True negatives are correctly called reference sites</li>" +
             "<li>Validated genotypes - are TP sites where the exact genotype (HET or HOM-VAR) has been validated </li> "     +
             "</ul>"+
+            "" +
+            "<h4>VCF Output:</h4>" +
+            "<ul>" +
+            "<li>The concordance state will be stored in the \"ST\" tag in the INFO field.</li>" +
+            "<li>If the truth and call sample names match, the call will have the \"_CALL\" suffix and the truth will have the " +
```

Comment:
@kbergin it's in a more recent commit, can you work off of that for your review?